packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -212,6 +213,33 @@ describe("get attribute(s)", () => {
             expect(attributeIds).toContain(relationshipAttributeId);
             expect(attributeIds).toStrictEqual(expect.arrayContaining(identityAttributeIds));
         });
+
+        test("should allow to get only default attributes", async function () {
```

Comment:
It would be cool if another RepositoryAttribute of type `Surname`, for example, would occur in that test, so that it is demonstrated that this RepositoryAttribute is not a default RepositoryAttribute as the other RepositoryAttribute of type `Surname` already is.

This RepositoryAttribute is then found in the test below as a RepositoryAttribute which is not a default Attribute.